tests/integration/targets/azure_rm_containerinstance/tasks/main.yml

```diff
@@ -143,6 +143,95 @@
       - output.containerinstances[1]['ports'] != None
       - output.containerinstances[1]['containers'] != None
 
+- name: Create sample container instance with volume
+  azure_rm_containerinstance:
+    resource_group: "{{ resource_group }}"
+    name: "aci{{ resource_group | hash('md5') | truncate(7, True, '') }}"
+    os_type: linux
+    ip_address: public
+    location: eastus
+    ports:
+      - 80
+    containers:
+      - name: mycontainer1
+        image: mcr.microsoft.com/azuredocs/aci-helloworld
+        memory: 1.5
+        volume_mounts:
+          - name: "my-filesharevolume"
+            mount_path: "/data/files"
+        ports:
+          - 80
+          - 81
+    volumes:
+      - name: "my-filesharevolume"
+        azure_file:
+          storage_account_name: "{{ storage_account_name }}"
```

Comment:
@aparna-patil Here you refer to a storage account, but you did not create and define it. Thank you very much!

Reply:
yes, @Fred-sun right! Actually this testcase refers to storage account and azure file share (on line no. 169). I tried adding test cases for storage account creation and passed it as reference further but currently we do not have any Ansible module for azure file share creation. So, Azure file share needs to be created manually before we execute the test case hence the storage account. Therefore, I have not added tasks for creating storage account. 

While testing locally, I am passing below values from integration_config.yml
```
---
resource_group: MyAzureResourceGroup
storage_account_name: mystorageaccount
storage_account_key: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
file_share_name: my-fileshare
```

Reply:
@resmo Got it! I will check it!